src/pages/organisation/index.tsx

```diff
@@ -286,9 +286,9 @@ const HomePage = () => (
             </p>
           </InnerText>
           <DetailsAndImage
-            imageUrl="/about/images/organisation-17.jpg"
+            imageUrl="/about/images/reader-funding.png"
```

Comment:
More of an observation/question, do we use png/jpeg's here instead of Grid images for readability and not accessed as frequently?